packages/truffle-code-utils/index.js

```diff
@@ -4,32 +4,56 @@ module.exports = {
   /**
    * parseCode - return a list of instructions given a 0x-prefixed code string.
    *
-   * Any contract metadata, if present, will be stripped.
+   * If numInstructions is not passed in, we attempt to strip contract
+   * metadata.  This won't work very well if the code is for a constructor or a
+   * contract that can create other contracts, but it's better than nothing.
+   *
+   * WARNING: Don't invoke the function that way if you're dealing with a
+   * constructor with arguments attached!  Then you could get disaster!
+   *
+   * If you pass in numInstructions (hint: count the semicolons in the source
+   * map, then add one) this is used to exclude metadata instead.
    *
    * @param  {String} hex_string Hex string representing the code
    * @return Array               Array of instructions
    */
-  parseCode: function (hex_string) {
-    // Convert to an array of bytes (denoted by substrings)
-    var code = hex_string.match(/(..?)/g);
+  parseCode: function(hex_string, numInstructions = null) {
```

Comment:
wanna change it to `hexString` while you're at it?

Reply:
Sure, why not. :P